tests/Classes/PublicReadonlyPropertiesWithConstructor.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brick\VarExporter\Tests\Classes;
+
+class PublicReadonlyPropertiesWithConstructor
```

Comment:
Shouldn't this be:

```suggestion
class ReadonlyPropertiesWithConstructor
```

As you have private readonly as well?